experimenter/tests/integration/nimbus/android/conftest.py

```diff
@@ -0,0 +1,154 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import os
+import subprocess
+import time
+from pathlib import Path
+
+import pytest
+import requests
+
+from .gradlewbuild import GradlewBuild
+from .models.models import TelemetryModel
+
+here = Path().cwd()
+
+
+def start_process(path, command):
```

Comment:
type annotations? :)

Reply:
I don't think we've used type annotations in any of our existing integration tests?

Reply:
Soon ™️